### PR TITLE
Style queue page

### PR DESCRIPTION
**Pull request by 0x213F**

### CODE REVIEW

**Comment by 0x213F** [comment]
@op1346 I ended up styling the queue a bit. Does this conflict with your current style changes? If no, can I merge this in?

File: src/components/QueueApp/QueueApp.module.css [hunk 1]
```diff
@@ -1,13 +1,28 @@
 .QueueApp {
+  font-family: Raleway;
```
Comment (op1346) [inline src/components/QueueApp/QueueApp.module.css:2]: Does the font not import from the index.css? Just wondering.

**Review by op1346 — APPROVED** [review]
Looks good!

File: src/components/QueueApp/QueueApp.module.css (hunk 1 above)
Comment (0x213F) [reply src/components/QueueApp/QueueApp.module.css:2]: Great question. I noticed that Raleway was not rendering as expected, so this was my band-aid approach to fixing the problem.

I took a closer look and found out what the problem was. Tailwind is overriding Raleway as the default font-style. The proper approach is to change the tailwind default

https://tailwindcss.com/docs/font-family

Which I did here (in #89)

https://github.com/0x213F/jukebox-radio-react/pull/89/commits/4883e2178f9d5533442a8a5940893af2ff8a76a3

Comment (op1346) [reply src/components/QueueApp/QueueApp.module.css:2]: Oh I see! Makes more sense.

**Comment by op1346** [comment]
> @op1346 I ended up styling the queue a bit. Does this conflict with your current style changes? If no, can I merge this in?

Oh man I didn't see this! Sorry!

**Pull request merged**